packages/opentelemetry-tracer-web/webpack/test.config.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This is the webpack configuration for browesr Karma tests.
+module.exports = {
+  mode: 'development',
+  target: 'web',
+  output: {filename: 'bundle.js'},
+  resolve: {extensions: ['.ts', '.js']},
+  devtool: 'inline-source-map',
+  module: {
```

Comment:
Could you add explicit config here (and in the other `test.config.js` file for basic-tracer) an explicit opt-out of most Node polyfills? See https://github.com/open-telemetry/opentelemetry-js/blob/a0140eca2265f21bc25f57b1413c2ee20fccba80/packages/opentelemetry-core/webpack/test.config.js#L30

I'd like to avoid polyfilling Node functionality and instead use our own lightweight polyfills for two reasons:
1. The polyfills provided by webpack will likely introduce a lot of extra code we don't really need, e.g. we just need a lightweight `hrtime` for the web, not a full `process` polyfill with other code. Yes, tree-shaking helps with this, but let's also do what we can so we don't need to rely on optimizations to trim code size.
2. Not everyone may build with webpack, and other builders / build tools may not bring in Node polyfills as easily

Reply:
Thanks for doing this, feel free to mark this as resolved!